examples/shallow/execute.py

```diff
@@ -94,7 +94,7 @@ def execute(option):
                                     stdout=subprocess.PIPE)
     if make_process.wait() != 0:
         return False, []
-    _ = make_process.stderr.read()
+    # _ = make_process.stderr.read()
```

Comment:
Was this because of landscape whining about underscored or unused variables. In some of my codes I've got around this by using subprocess.communicate() and returning stdout and stderr into a tuple. This way you can simply use stdout and it not complain, but then stderr is there should you need it later. Not sure if that is useful for you though